edb/schema/expraliases.py

```diff
@@ -33,17 +33,20 @@
 
 
 if TYPE_CHECKING:
+    from . import objects as so
     from . import schema as s_schema
 
 
-class AliasCommandContext(sd.ObjectCommandContext,
-                          s_anno.AnnotationSubjectCommandContext):
+class AliasCommandContext(
+    sd.ObjectCommandContext[Any],
+    s_anno.AnnotationSubjectCommandContext
+):
     pass
 
 
 class AliasCommand(
-    sd.QualifiedObjectCommand[s_types.Type],
-    s_types.TypeCommand,
+    sd.QualifiedObjectCommand[so.QualifiedObject],
```

Comment:
This was annotated as `sd.QualifiedObjectCommand[s_types.Type]`, but mypy complains: `Type argument "edb.schema.types.Type" of "QualifiedObjectCommand" must be a subtype of "edb.schema.objects.QualifiedObject"`.
I understand so.QualifiedObject is the right type argument here.

Reply:
Use `InheritingType`